lib/storage/storage.go

```diff
@@ -552,7 +554,7 @@ func (m *Metrics) Reset() {
 
 // UpdateMetrics updates m with metrics from s.
 func (s *Storage) UpdateMetrics(m *Metrics) {
-	m.RowsAddedTotal = rowsAddedTotal.Load()
+	m.RowsAddedTotal += s.rowsAddedTotal.Load()
```

Comment:
There is no need at increment. Just use assigned as it was before.

Reply:
Currently the increment doesn't make a difference indeed. But it will when the support for multiple storage objects is added. I.e. vmstorage will accept multiple -storageDataPath and for each will create a separate Storage object and then call UpdateMetrics() for each object. This is something we've been talking with @valyala for some time now.